solutions/usaco-993.mdx

```diff
@@ -0,0 +1,82 @@
+---
+id: usaco-993
+source: USACO Gold 2020 January
+title: Time is Mooney
+author: Nathan Gong
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_time_gold_jan20.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(T_{\text{max}} \cdot (N + M))$
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class TimeIsMooney {
+	public static void main(String[] args) throws IOException {
+		Scanner sc = new Scanner(new File("time.in"));
```

Comment:
scanner?

Reply:
i'll change it to kattio since everyone hates scanner 😢

Reply:
I'll change it to kattio since everyone hates scanner 😢

Reply:
I'll change it to kattio since everyone hates scanner 😢